domains/jvideo.json

```
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "jvideo-sk",
+           "email": "davedrives7@gmail.com",
+           "discord": "1030204181207859260"
+        },
+    
+        "record": {
+            "CNAME": "https://jvideo-sk.github.io"
```

Just noticed the problem :p 
```suggestion
            "CNAME": "jvideo-sk.github.io"
```